.github/workflows/ci.yml

```diff
@@ -0,0 +1,110 @@
+name: CI
+
+on:
+  # Run this on pushes to main
+  push:
+    branches:
+    - main
+
+  # Or when PR operations are done
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+
+jobs:
+  linter:
+    name: Run Helm Linter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.3.0
+
+      - name: Run linter
+        run: helm lint --set dataKey=BLOB ./conjur-oss
+
+  install-test-helm-v3:
+    name: Install/test Conjur with Helm V3 on KinD Cluster
+    needs: [ linter ]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        kube-tag:
+          - v1.18.2
+          - v1.16.9
+          - v1.14.10
+          - v1.12.10
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.2.1
+
+      - name: Create k8s KinD Cluster
+        uses: helm/kind-action@v1.0.0-rc.1
```

Comment:
We need to change this to the following to avoid the node taints that we're seeing in test results:
```
        uses: helm/kind-action@v1.2.0
```